sources/packages/backend/apps/queue-consumers/src/processors/schedulers/atbc-respone-integration/atbc-response-integration.scheduler.ts

```diff
@@ -0,0 +1,40 @@
+import { InjectQueue, Process, Processor } from "@nestjs/bull";
+import { Job, Queue } from "bull";
+import { BaseScheduler } from "../base-scheduler";
+import { QueueNames } from "@sims/utilities";
+import { QueueService } from "@sims/services/queue";
+import { ProcessPDRequestQueueOutDTO } from "./models/atbc-response-integration.dto";
+import { ATBCIntegrationProcessingService } from "@sims/integrations/atbc-integration";
+
+/**
+ * Process all the applied PD requests to verify the status with ATBC.
+ */
+@Processor(QueueNames.ATBCResponseIntegration)
+export class ATBCResponseIntegrationScheduler extends BaseScheduler<void> {
+  constructor(
+    @InjectQueue(QueueNames.ATBCResponseIntegration)
+    protected readonly schedulerQueue: Queue<void>,
```

Comment:
Why `schedulerQueue` is declared as `protected` here?

Reply:
Totally makes sense, it was missed .